### PR TITLE
 Updated models.go and completed the mentorReg function

### PR DESCRIPTION
**Pull request by spookbite**

### CODE REVIEW

File: routes/mentor.go [hunk 1]
```diff
@@ -3,8 +3,42 @@ package routes
...
+	err := json.Unmarshal(body, &mentor)
+	if err != nil {
+	  http.Error(w, err.Error(), 400)
+	  return 		
+	}
+
+	
+	db, err := gorm.Open("sqlite3", "kwoc.db")
+	if err != nil {
+		 http.Error(w, err.Error(), 500)
```
Comment (rakaar) [inline routes/mentor.go:25]: bro, we discussed that we would keep it as w.Write naa

File: routes/mentor.go [hunk 2]
```diff
@@ -3,8 +3,42 @@ package routes
...
+		 return
+	}
+	mentor.db = db
```
Comment (rakaar) [inline routes/mentor.go:28]: I dint get this?
directly db.Create is enuf right?

Comment (spookbite) [reply routes/mentor.go]: I thought this can also be done. So, tried it once

**Comment by grapheo12** [comment]
@spookbite Squash into a single commit and put a good commit message while doing so. Then force push your branch.
This will help make a clean Git history.

**Comment by grapheo12** [comment]
LGTM! Merging. Great work @spookbite .

**Pull request merged**